packages/react-toast/package.json

```diff
@@ -21,9 +21,11 @@
     "types": "tsc --declaration --outDir typings/ --emitDeclarationOnly --declarationMap --allowJs false --checkJs false"
   },
   "license": "MIT",
+  "devDependencies": {
+    "concat-md": "^0.3.5"
+  },
   "dependencies": {
-    "@trycourier/react-provider": "^0.1.12-beta.0",
-    "concat-md": "^0.3.5",
+    "@trycourier/react-provider": "^0.1.12-beta.1",
```

Comment:
did we do `yarn install`? I would expect some changes to lock files no?

Reply:
Yep, no changes to locks